CHANGES.rst

```diff
@@ -8,6 +8,7 @@ Version 0.7.3 (UNRELEASED)
 - Fixes HTCondor Docker networking and machine version requirement setup.
 - Fixes HTCondor logs and workspace files retrieval on job failure.
 - Fixes Slurm job submission providing the correct shell environment to run Singularity.
+- Fixes Slurm job command dump to take into account complex shell constructs.
```

Comment:
What about: Fixes Slurm job command generation for cases of complex shell expressions.

BTW, in the commit log, `s/jog/job/g`.